llvm/lib/Analysis/Delinearization.cpp

```diff
@@ -182,7 +182,7 @@ void llvm::collectParametricTerms(ScalarEvolution &SE, const SCEV *Expr,
   LLVM_DEBUG({
     dbgs() << "Strides:\n";
     for (const SCEV *S : Strides)
-      dbgs() << *S << "\n";
+      dbgs() << "  " << *S << "\n";
```

Comment:
```suggestion
      dbgs().indent(2) << *S << "\n";
```

The current one is fine if you prefer it. Same as other parts.